libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -274,6 +275,9 @@ AP_BattMonitor::init()
             case Type::FuelLevel_PWM:
                 drivers[instance] = new AP_BattMonitor_FuelLevel_PWM(*this, state[instance], _params[instance]);
                 break;
+            case Type::FuelLevel_Analog:
```

Comment:
#if AP_BATTMON_SMBUS_ENABLE

Reply:
Macros created for each of the fuel features. All gated off from 1MB boards. Added the options to the build server too.